docs/quickstart.md

```diff
@@ -0,0 +1,249 @@
+
+Once the post deployment script has completed, the terminal prompt will change to the GitHub username followed by the name of the lab.
+
+In the screenshot below, the GitHub username is `MitchV85` and the lab is `techlib-vxlan-domain-a`.
+
+<figure markdown>
+![Post Deployment Script](assets/img/aclabs-quickstart-postdeploy-complete.png "Post Deploy"){ width=800px }
+<figcaption>Post Deploy Script Complete (Click to Zoom)</figcaption>
+</figure>
+
+At this point, the lab is ready to go! Running the following command in the terminal will provide an overview and status of all deployed nodes:
+
+```bash
+make inspect
+```
+
+<figure markdown>
+![Post Deployment Script](assets/img/aclabs-quickstart-make-inspect.png "Post Deploy"){ width=800px }
+<figcaption>Lab Status Output (Click to Zoom)</figcaption>
+</figure>
+
+## Interacting with the Lab
+
+### SSH
+
+Once the lab is up and running, we can use the terminal in our Codespace to SSH into the nodes.
+
+??? tip "More real estate for those SSH sessions 🏠"
+    Increase the size of the terminal by selecting the `Maximize Panel Size` button to the right of the terminal:
+
+    <figure markdown>
+    ![Post Deployment Script](assets/img/aclabs-quickstart-maximize-panel.png "Post Deploy"){ width=800px }
+    <figcaption>Maximize the Terminal (Click to Zoom)</figcaption>
+    </figure>
+
+??? tip "Use tabs 📑"
+    Create a new tab for an SSH session by selecting `New Terminal` to the right of the terminal.
+
+    Alternatively, the following keyboard shortcut can be used to open a new tab on both Windows and macOS: ++ctrl+shift+single-quote++
+
+    Once opened, the new terminal tab can be renamed by right-clicking on the tab and selecting `Rename`
+
+    === "New Terminal"
+        <figure markdown>
+        ![Post Deployment Script](assets/img/aclabs-quickstart-new-tab.png "Post Deploy"){ width=800px }
+        <figcaption>New Terminal Tab (Click to Zoom)</figcaption>
+        </figure>
+
+    === "Rename the Tab"
+        <figure markdown>
+        ![Post Deployment Script](assets/img/aclabs-quickstart-rename-tab.png "Post Deploy"){ width=800px }
+        <figcaption>Rename the Terminal Tab (Click to Zoom)</figcaption>
+        </figure>
+
+    === "Party Time 🎉"
+        <figure markdown>
+        ![Post Deployment Script](assets/img/aclabs-quickstart-renamed-tabs.png "Post Deploy"){ width=800px }
+        <figcaption>Rename the Terminal Tab (Click to Zoom)</figcaption>
+        </figure>
+
+A list of `Lab Hosts` that are accessible via `SSH` from within the codespace can be viewed at any time from the terminal by entering the following command:
+
+```bash
+make inspect
+```
+
+To access a node, use the `ssh` command followed by `admin@hostname` as shown in the example below
+
+```bash
+ssh admin@A-SPINE1
+```
+
+??? question "What's the password? :lock:"
+    Unless stated otherwise, the default username of `admin` and password of `admin` is used for all nodes in an Arista Community lab
```

Comment:
I'd rather write that passwords will be provide in the lab documentation. Don't like the idea of setting the default even if settle on that. The reason: avd container has avd user by default. To avoid entering the login it will be provisioned for most AVD labs.

Reply:
Agreed. Note has been updated.